app/src/main/java/io/intrepid/contest/screens/contestcreation/describecontest/DescribeContestPresenter.java

```diff
@@ -40,9 +39,14 @@ public void onNextValidated() {
         view.setNextEnabled(true);
     }
 
+    boolean isEmpty(CharSequence sequence) {
+        String sequenceString = String.valueOf(sequence);
+        return sequenceString.trim().isEmpty();
```

Comment:
I understand you did this to be able to test without a static Utils method... but adding this method in multiple presenters makes me confused.
I don't think you need `trim()` eihter; to make it simple I'd just add to `onTextChanged`:
```
if (newDescription.toString().isEmpty()) {
...
}
```